.github/workflows/main.yml

```diff
@@ -17,24 +17,14 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
         with:
-          java-version: 11
+          java-version: 17
+          distribution: temurin
 
-      - name: Cache
-        uses: actions/cache@v2.0.0
-        with:
-          # Cache gradle directories
-          path: |
-            ~/.gradle/caches/jars-3
-            ~/.gradle/caches/modules-2/files-2.1/
-            ~/.gradle/caches/modules-2/metadata-2.96/
-            ~/.gradle/native
-            ~/.gradle/wrapper
-          # Key for restoring and saving the cache
-          key: ${{ runner.os }}-gradle
+      - uses: gradle/gradle-build-action@v2
```

Comment:
is linting automatically checked when using `./gradlew check`?

Reply:
Yes indeed. Explicitly check with `ktlintCheck` and automatically fix with `ktlintFormat`.